purchases/src/main/kotlin/com/revenuecat/purchases/common/Backend.kt

```diff
@@ -656,6 +662,57 @@ internal class Backend(
         }
     }
 
+    fun postRedeemRCBillingPurchase(
+        appUserID: String,
+        redemptionToken: String,
+        onSuccessHandler: () -> Unit,
+        onErrorHandler: (PurchasesError) -> Unit,
+    ) {
+        val endpoint = Endpoint.PostRedeemRCBillingPurchase(appUserID)
+        val path = endpoint.getPath()
+        val body = mapOf("new_app_user_id" to redemptionToken) // WIP: change this field to the actual field
```

Comment:
We will create a new endpoint for this, instead of relying on the aliasing endpoint. For now, reusing that one and we need to use the corresponding parameter names.